apps/ejabberd/src/mod_muc_light.erl

```diff
@@ -72,7 +72,8 @@
          is_room_owner/3,
          can_access_room/3,
          can_access_identity/3,
-         muc_room_pid/2]).
+         muc_room_pid/2,
+         delete_room/1]).
```

Comment:
It's not a hook handler. It should be a part of export for administration API.

Reply:
Sure Piotr. This now moved to the administration API.